src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/TestHelper.cs

```diff
@@ -12,6 +13,12 @@ public static bool AreEqual(object obj1, object obj2)
             return result.AreEqual;
         }
 
+        public static bool EnumerablesAreEqual<T>(IEnumerable<T> expected, IEnumerable<T> actual)
```

Comment:
Another option to this is using FluentAssertions which has a `listA.Should().AllBeEquivalentTo(listB)` or similar. Suggestion only.

Reply:
Personally, I prefer the standard Asserts over FluentAssertions.

The BeEquivalentTo method in FluentAssertions compares the elements using their Object.Equals method. That means that (in general)
1) you have to implement the Equals and GetHashCode methods on the element's type especially for the BeEquivalentTo call (if they are not already implemented)
2) it doesn't handle the case where the types are different but comparable

In this particular case, the types are KeyValuePair structs, so BeEquivalentTo would work fine.
When looking at the test, I did notice that the Assert.IsTrue was missing, so I've added that.